libraries/protocol/steem_operations.cpp

```diff
@@ -76,13 +76,42 @@ namespace steemit { namespace protocol {
       }
    }
 
+   struct comment_options_extension_validate_visitor
+   {
+      comment_options_extension_validate_visitor() {}
+
+      typedef void result_type;
+
+      void operator()( const comment_payout_beneficiaries& cpb ) const
+      {
+         cpb.validate();
+      }
+   };
+
+   void comment_payout_beneficiaries::validate()const
+   {
+      uint32_t sum = 0;
+
+      FC_ASSERT( beneficiaries.size() < 128, "Cannot specify more than 127 beneficiaries." ); // Require size serializtion fits in one byte.
+
+      for( auto& route : beneficiaries )
```

Comment:
ensure they are sorted by weight / beneficiary and that each beneficiary is only included once